tests/components/nut/test_config_flow.py

```diff
@@ -95,6 +180,20 @@ def _get_mock_pynutclient(list_vars=None):
     assert len(mock_setup_entry.mock_calls) == 1
 
 
+async def test_form_import_dupe(hass):
+    """Test we get abort on duplicate import."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+
+    entry = MockConfigEntry(domain=DOMAIN, data=VALID_CONFIG)
+    entry.add_to_hass(hass)
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": "import"}, data=VALID_CONFIG
+    )
+    assert result["type"] == "abort"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5329694)